backend/src/main/java/com/woowacourse/gongseek/member/application/MemberService.java

```diff
@@ -0,0 +1,68 @@
+package com.woowacourse.gongseek.member.application;
+
+import com.woowacourse.gongseek.article.domain.Article;
+import com.woowacourse.gongseek.article.domain.repository.ArticleRepository;
+import com.woowacourse.gongseek.auth.presentation.dto.AppMember;
+import com.woowacourse.gongseek.comment.domain.Comment;
+import com.woowacourse.gongseek.comment.domain.repository.CommentRepository;
+import com.woowacourse.gongseek.member.domain.Member;
+import com.woowacourse.gongseek.member.domain.repository.MemberRepository;
+import com.woowacourse.gongseek.member.exception.MemberNotFoundException;
+import com.woowacourse.gongseek.member.presentation.dto.MyPageArticleResponse;
+import com.woowacourse.gongseek.member.presentation.dto.MyPageArticlesResponse;
+import com.woowacourse.gongseek.member.presentation.dto.MyPageCommentResponse;
+import com.woowacourse.gongseek.member.presentation.dto.MyPageCommentsResponse;
+import com.woowacourse.gongseek.member.presentation.dto.MemberDto;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class MemberService {
+
+    private final MemberRepository memberRepository;
+
+    private final ArticleRepository articleRepository;
+
+    private final CommentRepository commentRepository;
+
+    public MemberDto getOne(AppMember appMember) {
+        Member member = getMember(appMember);
+        return new MemberDto(member);
+    }
```

Comment:
```suggestion
    @Transactional(readOnly = true)
    public MemberDto getOne(AppMember appMember) {
        Member member = getMember(appMember);
        return new MemberDto(member);
    }
```
조회 로직에 readOnly 옵션을 추가해주세요.
아래 메서드도 마찬가지입니다.

Reply:
3개 메서드 모두 조회용 메서드라서 클래스 레벨에 추가해주었습니다!